include/my_crypt.h

```diff
@@ -0,0 +1,69 @@
+/*
+ Copyright (c) 2014 Google Inc.
+ Copyright (c) 2014, 2015 MariaDB Corporation
+
+ This program is free software; you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation; version 2 of the License.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program; if not, write to the Free Software
+ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301  USA */
+
+#ifndef MY_CRYPT_INCLUDED
+#define MY_CRYPT_INCLUDED
+
+#include <my_config.h> /* HAVE_EncryptAes128{Ctr,Gcm} */
+#include "my_aes.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define MY_AES_OK                     0
```

Comment:
Consider converting to enum and return that enum from functions instead of int

Reply:
It is following the convention used in my_aes.h, were all values returned from my_aes functions are defined ints.